r37932 | al | 2017-04-20 01:06:04 +0200 (Thu, 20 Apr 2017) | 6 lines

vo_vdpau: Explicitly include header vdpau_x11.h

The include from inside libavcodec/vdpau.h was removed in FFmpeg
commit d40e181bec22014a9ea312ab6837f7f0bc4f9e42 .

Index: libvo/vo_vdpau.c
===================================================================
--- a/libvo/vo_vdpau.c	(revision 37931)
+++ b/libvo/vo_vdpau.c	(revision 37932)
@@ -34,6 +34,7 @@
 
 #include <stdio.h>
 #include <strings.h>
+#include <vdpau/vdpau_x11.h>
 
 #include "config.h"
 #include "sub/ass_mp.h"
